spec/workers/build_index_page_worker_spec.rb

```diff
@@ -1,28 +1,59 @@
 RSpec.describe BuildIndexPageWorker, type: :worker do
   describe '#methods' do
     describe 'build index page' do
-      let(:commodity) do
-        create :commodity, :with_description, description: 'test description'
+      before do
+        # Make sure index is fresh
+        TradeTariffBackend.search_client.drop_index(search_index)
+        TradeTariffBackend.search_client.create_index(search_index)
+        commodities # trigger creation of commodity
+
+        described_class.new.perform 'search', search_index.name_without_namespace, 1, 10
+      end
+
+      let(:commodities) do
+        create_pair :commodity, :with_description, description: 'test description'
+      end
+
+      let(:search_index) { Search::CommodityIndex.new }
+
+      let(:search_result) do
+        TradeTariffBackend.search_client.search q: 'test', index: search_index.name
+      end
+
+      it 'bulk indexes all model entries' do
+        expect(search_result.hits.total.value).to be >= 1
       end
 
+      it 'has bulk indexed the expected commodity' do
+        expect(search_result.hits.hits.map(&:_source).map(&:goods_nomenclature_item_id)).to \
```

Comment:
we could improve readability here,
I guess we can simplify with some var with meaningful names 🤔

Reply:
Yeh, good call - I'll move it into a `let` to give it a name